### PR TITLE
feat: bump Linux headers to match the Linux kernel in pkgs

### PR DESCRIPTION
**Pull request by smira**
This should ensure that software we're building is able to use the most
recent Linux features.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

File: linux-headers/pkg.yaml [hunk 1]
```diff
@@ -34,11 +34,13 @@ steps:
             ;;
         esac
 
-        make -j $(nproc) INSTALL_HDR_PATH=dest headers_install ARCH=${arch}
+        make -j $(nproc) headers ARCH=${arch}
```
Comment (smira) [inline linux-headers/pkg.yaml:37]: headers_install now relies on `rsync` which we don't have: http://www.linuxfromscratch.org/lfs/view/development/chapter06/linux-headers.html

Comment (andrewrynhard) [reply linux-headers/pkg.yaml:37]: Is this a workaround for 'installing' it?

**Pull request merged**